docker/generic/start_proxy.py

```diff
@@ -433,6 +436,15 @@ def make_argparser():
         help='''
         comma separated outgoing trace contexts (traceparent|grpc-trace-bin|x-cloud-trace-context)'''
     )
+    parser.add_argument(
+        '--cloud_trace_url_override',
+        default="",
+        help='''
+        By default, traces will be sent to production Stackdriver Tracing.
+        If this is non-empty, will send traces here instead.
+        Must be in gRPC format: https://github.com/grpc/grpc/blob/master/doc/naming.md
```

Comment:
This name must point to a gRPC service (please add a link to the service proto definition),  The name must be in gRPC format: https://github.com/grpc/grpc/blob/master/doc/naming.md

Reply:
Could you also update the help text for config_mgr flag --tracing_stackdriver_address

Reply:
Done. Did not include URLs in config manager flags, but included them in `start_proxy.py`.